internal/commands/push/command.go

```diff
@@ -259,12 +259,14 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		return nil
 	}
 
-	proceed, err := ui.Confirm("Please confirm the changes shown above")
-	if err != nil {
-		return err
-	}
-	if !proceed {
-		return nil
+	if !isNewApp {
```

Comment:
I think this confirmation is still valuable to have.. assuming we're not showing an empty diff

was there a particular reason we chose to skip this now in the event we're creating a new app?

Reply:
Discussed on Slack - for reference, we've decided instead to log the diff (which we weren't doing when creating a new app), while including the additional prompting to confirm app config.